### PR TITLE
Rsww 13 and 10 - Transport and options

### PR DESCRIPTION
**Pull request by Danzigerrr**
added:
-     more detailed fields in Tour class
-     food options, restaurant details
-     images of hotel
-     transport options (list of available airports)

### CODE REVIEW

**Review by pcichowski — COMMENTED** [review]
maybe I'm doing something wrong, but I can't get it to work out of the box:

```
Traceback (most recent call last):
  File "/home/silentsky/uni/rsww/Travel-Agencies-Scrapers/Itaka scraper/main.py", line 8, in <module>
    offers_list = scrape_all_inclusive_offers(driver)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/silentsky/uni/rsww/Travel-Agencies-Scrapers/Itaka scraper/scrape_links_to_offers.py", line 20, in scrape_all_inclusive_offers
    new_tour = tour(tour_web_link)
               ^^^^^^^^^^^^^^^^^^^
TypeError: 'WebElement' object is not callable
```

File: Itaka scraper/scrape_links_to_offers.py [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
-        # save all offers
-        for offer in offers_on_website:
-            offer_web_link = offer.get_attribute('href')
-            new_offer = Offer(offer_web_link)
-            offers.append(new_offer)
+        # save all tours
+        for tour in tours_on_website:
+            tour_web_link = tour.get_attribute('href')
+            new_tour = tour(tour_web_link)
```
Comment (pcichowski) [inline Itaka scraper/scrape_links_to_offers.py]: I reckon this should be 
```suggestion
            new_tour = Tour(tour_web_link)
```

**Review by pcichowski — COMMENTED** [review]
Adding types to functions and/or some brief documentation could be vital, like so:
```python
from typing import List

def scrape_all_inclusive_offers(driver) -> List[Tour]:
```

File: Itaka scraper/scrape_links_to_offers.py (hunk 1 above)
Comment (Danzigerrr) [reply Itaka scraper/scrape_links_to_offers.py]: That's right. I am fixing some other bugs now

**Comment by Danzigerrr** [comment]
@pcichowski @Mixss review the new code please

**Comment by Mixss** [comment]
Still doesn't work. 
image

**Comment by Danzigerrr** [comment]
@Mixss share me the link which was on the console just before the error occured

**Comment by Mixss** [comment]
Appeove

**Pull request merged**